content/en/docs/languages/java/instrumentation.md

```diff
@@ -191,10 +191,10 @@ While the [LoggerProvider](../api-components/#loggerprovider) /
 equivalent [trace](../api-components/#tracerprovider) and
 [metric](../api-components/#meterprovider) APIs, they serve a different use
 case. As of now, `LoggerProvider` / `Logger` and associated classes represent
-the [Log Bridge API](/docs/specs/otel/logs/bridge-api/), which exists to write
-log appenders to bridge logs recorded through other log APIs / frameworks into
-OpenTelemetry. They are not intended for end user use as a replacement for Log4j
-/ SLF4J / Logback / etc.
+the [Log API](/docs/specs/otel/logs/api/), which exists to write log appenders
+to bridge logs recorded through other log APIs / frameworks into OpenTelemetry.
```

Comment:
the new (experimental) Logs API has a broader scope than existing to write log appenders

Reply:
Change reverted.